Sources/SQLiteDriver.swift

```diff
@@ -2,7 +2,15 @@ import Fluent
 
 public class SQLiteDriver: Fluent.Driver {
 
-	let database = SQLite()
+	var database = SQLite()
+
+	public init() {
+		database.initWithPath("Database/main.sqlite")
+	}
+
+	public init(path: String){
+		database.initWithPath(path)
```

Comment:
I could see someone wanting to share a database between a Vapor app and possibly a Ruby/Java app, so we should let this path be anywhere on the system.

But to make this work with Vapor providers I think we need to have a static variable on the driver like `SQLiteDriver.databasePath = "/path/to/db/main.sqlite"`